pkg/dynamicaccess/container.go

```diff
@@ -0,0 +1,7 @@
+package dynamicaccess
+
+// iterator
+type Container interface {
+	Add(oldItemKey string, oldRootHash string) (newRootHash string, err error)
+	Get(rootKey string) (value string, err error)
+}
```

Comment:
`container.go` and `act.go` should be one file.

Reply:
Instead of act.go, a key-value store package should exist, like 'index' package, or 'map' package.

// Viktor